java/src/main/java/com/avatao/tfw/tfwconnector/Utilities.java

```diff
@@ -0,0 +1,30 @@
+package com.avatao.tfw.tfwconnector;
+
+import com.avatao.tfw.tfwconnector.TFWServerConnector;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+
+public class Utilities {
+    private TFWServerConnector serverConnector;
+
+    public Utilities(TFWServerConnector serverConnector) {
```

Comment:
There should be another constructor which requires no parameters and initializes a `TFWServerConnector` internally.